setup.py

```diff
@@ -0,0 +1,28 @@
+from setuptools import find_packages, setup
+
+
+setup(
+    version='0.1.0',
+    name='TODO_PROJECT_NAME',
+    packages=find_packages(),
+    include_package_data=True,
+    install_requires=[],
+    description='TODO',
+    author='Incuna Ltd',
+    author_email='admin@incuna.com',
+    url='https://github.com/incuna/TODO_PROJECT_NAME',
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
```

Comment:
I don't know what the other options are but if you have a more suitable suggestion I'd be happy to go with it.